packages/strapi-utils/lib/models.js

```diff
@@ -268,12 +268,17 @@ module.exports = {
     }
 
     const model = entity.toLowerCase();
+    let models = _.clone(strapi.models);
```

Comment:
```js
const models = _.assign(_.clone(strapi.models), Object.keys(strapi.plugins).reduce((acc, current) => {
  _.assign(acc, _.get(strapi.plugins[current], ['models'], {}));
  return acc;
}, {}));
```

Use const instead of let?